routes/cds-services.js

```diff
@@ -417,6 +418,26 @@ function getFHIRClient(req, res) {
   }
 }
 
+function addDiagnosticReportToBundle(customApiResponse, bundle, client) {
+  const orders = customApiResponse.Order ?? [];
+  console.log(`Orders count: ${orders.length}`);
+  orders.forEach(order => {
+    const searchURL = `DiagnosticReport?identifier=${order.OrderId}`;
```

Comment:
Should we also filter by the patient ID here? Or were you thinking it doesn't matter since the `identifier` will be unique to each DiagnosticReport?